Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM php:8.2
+
+COPY . .
```

Comment:
```suggestion
```
Este COPY é desnecessário pois o volume já está sendo montado e neste caso trabalhar com volumes é mais interessante pois precisamos do código atualizado no container sempre que for alterado na máquina host sem precisar fazer build novamente do projeto e executar este copy.

Reply:
Pensando aqui, se remover o entrypoint não funciona.

O ideal então seria criar um arquivo entrypoint.sh, usar o COPY para pegar o entrypoint.sh e nele colocar a execução do entrypoint. Assim tudo o que é código fica em volume.

Final do arquivo `.docker/Dockerfile`
```dockerfile
COPY entrypoint.sh /var/www/scripts/
ENTRYPOINT [ "bash", "/var/www/scripts/entrypoint.sh" ]
```

`.docker/entrypoint.sh`
```bash
#!/bin/bash

php /var/www/html/vendor/bin/jigsaw serve --host=0.0.0.0
```

Neste caso talvez vale criar no path `.docker/entrypoint.sh`, mover o Dockerfile para dentro da pasta `.docker` e substituir no docker-compose o `build: .` por: `build: .docker`, assim tudo que é relativo ao container fica numa dotfile oculta.

Reply:
Gostei muito dessa abordagem aê!

Reply:
https://github.com/LibreCodeCoop/site/blob/master/.docker/php7/Dockerfile#L26-L27

https://github.com/LibreCodeCoop/site/blob/master/.docker/php7/entrypoint.sh

Neste exemplo aí é para Jigsaw também mas to rodando diretamente o comando `npm run watch` ao invés de fazer isto pelo comando do Jigsaw, no final das contas dá no mesmo.